translate/rus.json

```diff
@@ -344,13 +344,13 @@
 "show ply":"показать слой",
 "single line":"одна линия",
 "small decimal":"малые десятичные",
-"sound capture":"звук захвата",
+"sound capture":"звук взятия",
 "sound check":"звук чека",
 "sound checkmate":"звук мат",
-"sound draw":"звук ничьей",
+"sound draw":"звук при ничье",
```

Comment:
Lines 350 and 353: When I read the translation, I think that 'draw/win itself makes a sound', not 'this is a sound in case of draw/win'. The new translation fixes that.